### DISCUSSION TITLE
Set Intial App Size

### DISCUSSION THREAD

**Discussion by SirRuggie** [microsoft/WinUI-Gallery#1035] [Q&A] [answered]
How would I set the initial app size in WinUI 3?

**Comment by marcelwgn** [comment, marked as answer]
The approach I would use is to set the window size manually before showing the window. [This answer](https://github.com/microsoft/WinUI-Gallery/discussions/1056#discussioncomment-3286312) has more information on how to set the size of your app.

**Reply by HEIC-to-JPEG-Dev** [reply]
Hopefully in SDK 1.7 the windows will remember their position and size - this is expected behavour

**Reply by Poker-sang** [reply]
Although this can work. However, if the window is enlarged, the lower bound of the window may exceed the screen. I think it's still necessary to specify the size when creating the window
